setup.cfg

```diff
@@ -58,6 +58,8 @@ ignore =
   F403,
   # F405 '*' may be undefined, or defined from star imports: *
   F405
+  # line break before binary operator, black does this with pathlib.Path objects
+  W503
```

Comment:
You should probably now declare a minimum pip version as a runtime dependency, since you have pip module names hardcoded.

Reply:
> You should probably now declare a minimum pip version as a runtime dependency, since you have pip module names hardcoded.

If anything, I would think it'd be a max pip version? Since pip is installed at build time and shipped with relenv builds.

Reply:
Disallowing pip upgrades, although I see your point, not allowing pip to be upgraded might be seen as a bad security practice...
Setting a minimal also dissalows downgrades(for whatever reasons a user might have to do this).
I think the wheel part might be fairly stable, the one I see going away is the legacy approach...
If you think we'll have a fair amount of releases per year, then yeah, set a maximum.
If you see us with maybe just one or two releases a year(once we stabelize) that max might just force us to release so that we keep up with pip...

I don't know, I'll leave the max version at your discretion....

Reply:
At the moment I am inclined to punt on doing anything here and letting the bug reports roll in. Maybe just document that we only test with the version of pip shipped with relenv.

Reply:
Ok. Works for me.